source/reference/versioning.txt

```diff
@@ -82,6 +82,69 @@ released, version 2.0 would no longer receive patch releases. At the
 same time, version 1.3 would continue to receive patches until version
 1.4 was released.
 
+.. _c2c-versioning-live-upgrade:
+
+Live Upgrade
+~~~~~~~~~~~~
+
+.. versionadded:: 1.6.0
+
+Starting in ``mongosync`` 1.6.0, you can upgrade ``mongosync`` without
+restarting data synchronization operations from the beginning. This is
```

Comment:
we should make it clear that users can only live upgrade **from** mongosync 1.5 (or newer) to the latest mongosync version